diff --git a/src/main/org/apache/tools/ant/Project.java b/src/main/org/apache/tools/ant/Project.java
index 5b20ca0c2..1aa0688ea 100644
--- a/src/main/org/apache/tools/ant/Project.java
+++ b/src/main/org/apache/tools/ant/Project.java
@@ -62,7 +62,7 @@ import org.apache.tools.ant.util.VectorSet;
  * file paths at runtime.
  *
  */
-public class Project implements ResourceFactory {
+public class Project implements xResourceFactory {
     /** Message priority of &quot;error&quot;. */
     public static final int MSG_ERR = 0;
     /** Message priority of &quot;warning&quot;. */
